application/helpers/frontend_helper.php

```diff
@@ -2214,3 +2215,30 @@ function cookieConsentLocalization()
         gT('Please be patient until you are forwarded to the final URL.')
     );
 }
+
+/**
+ * Returns an array of URL parameters that can be forwarded
+ *
+ * @return array<string,mixed>
+ */
+function getForwardParameters()
+{
+    $reservedGetValues = array(
+        'token',
+        'sid',
+        'gid',
+        'qid',
+        'lang',
+        'newtest',
+        'action',
+        'seed',
+        'index.php',
+    );
+
+    $request = Yii::app()->getRequest();
```

Comment:
Can you move this as an argument to the function? Then we're gold!

Reply:
Ready for reviewal